crates/bevy_picking_input/src/inputs.rs

```diff
@@ -0,0 +1,43 @@
+use bevy::prelude::*;
+use bevy_picking_core::input::{CursorClick, CursorId, CursorMultiSelect};
+
+/// Unsurprising default picking inputs
+pub fn default_picking_inputs(
+    mouse: Res<Input<MouseButton>>,
+    keyboard: Res<Input<KeyCode>>,
+    touches: Res<Touches>,
+    mut cursor_query: Query<(&CursorId, &mut CursorMultiSelect, &mut CursorClick)>,
+) {
+    let is_multiselect_down = keyboard.any_pressed([
+        KeyCode::LControl,
+        KeyCode::RControl,
+        KeyCode::LShift,
+        KeyCode::RShift,
+    ]);
```

Comment:
This would likely benefit from having some layer of separation between the actions taken here (multiselect) and keyboard buttons. I'm sure you've considered that already and it would be a discussion about improving the bevy_input rather than doing anything here

Reply:
All the systems that provide default inputs are just that - defaults. The API is designed so you can trigger any input with anything you want. I'll be making an example with leafwing input manager.